impl/src/main/java/io/jsonwebtoken/impl/io/RuntimeClasspathSerializerLocator.java

```diff
@@ -34,6 +34,8 @@
             return Classes.newInstance("io.jsonwebtoken.io.JacksonSerializer");
         } else if (isAvailable("io.jsonwebtoken.io.OrgJsonSerializer")) {
             return Classes.newInstance("io.jsonwebtoken.io.OrgJsonSerializer");
+        } else if (isAvailable("com.google.gson.GsonBuilder")) {
```

Comment:
This should be checking for "io.jsonwebtoken.io.GsonDeserializer" not "com.google.gson.GsonBuilder"

Reply:
Ok patching.